server.js

```diff
@@ -22,12 +18,75 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Start defining routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+//get all films with the path "films"
+app.get('/films', (req, res) => {
+  res.json(netflixData)
+})
+
+//get film from specific year with for example the path "year/2019"
+app.get('/year/:year', (req, res) => {
+ const year = req.params.year
+ const filmsFromYear = netflixData.filter((item) => item.release_year === +year)
+ res.json(filmsFromYear)
+})
+
+//get specific film with for example the path "film/Chocolate"
+app.get('/film/:title', (req, res) => {
+  const title = req.params.title
+  const film = netflixData.filter((item) => item.title === title)
+  res.json(film)
+ })
+
+ //get the type of show with for example the path "type/Movie"
+app.get('/type/:type', (req, res) => {
+  const type = req.params.type
+  const typeOfFilm = netflixData.find((item) => item.type === type)
```

Comment:
this will only return a single object, so if there are more shows with the same type, you'll only get the first match.
If you want to return multiple shows, use filter() method instead.